isis/tests/CMakeLists.txt

```diff
@@ -3,13 +3,16 @@ cmake_minimum_required(VERSION 3.10)
 add_dependencies(isis3 isis3)
 
 file(GLOB test_source "${CMAKE_SOURCE_DIR}/tests/*.cpp")
-file(GLOB isis_libs "${CMAKE_BINARY_DIR}/lib/lib*")
+
+# Non-ideal temporary solution to link against other shared libraries for mission tests, etc
+file(GLOB isis_libs "${CMAKE_BINARY_DIR}/lib/*.so")
+file(GLOB isis_mac_libs "${CMAKE_BINARY_DIR}/lib/*.dylib")
 
 # Link runISISTests with what we want to test and the GTest and pthread library
 add_executable(runISISTests
                IsisTestMain.cpp
                ${test_source})
 
-target_link_libraries(runISISTests isis3 ${isis_libs} ${ALLLIBS} ${GTEST_LIBRARIES} ${GTEST_MAIN_LIBRARIES} Threads::Threads)
+target_link_libraries(runISISTests isis3 ${isis_libs} ${isis_mac_libs} ${ALLLIBS} ${GTEST_LIBRARIES} ${GTEST_MAIN_LIBRARIES} Threads::Threads)
```

Comment:
I realize one of the two lists of libs should always be empty, but why not put  *.so and *.dylib into one list (isis_libs) and not have the os specific isis_mac_lib even exist OR have a list populated based on the OS?
But is should work, so I'm ok with this too